services/movie.py

```diff
@@ -0,0 +1,43 @@
+from typing import List
+
+from django.db.models import QuerySet
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: List[int] = None,
+        actors_ids: List[int] = None
+) -> QuerySet:
+    if genres_ids is None and actors_ids is None:
+        return Movie.objects.all()
+    if genres_ids is not None and actors_ids is not None:
+        return Movie.objects.filter(
+            genres__id__in=genres_ids,
+            actors__id__in=actors_ids,
+        ).distinct()
+    if genres_ids is not None:
+        return Movie.objects.filter(genres__id__in=genres_ids).distinct()
+    if actors_ids is not None:
+        return Movie.objects.filter(actors__id__in=actors_ids).distinct()
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.filter(id=movie_id).first()
```

Comment:
The `get_movie_by_id` function uses `filter().first()` which returns `None` if no movie is found. Consider handling this case explicitly to provide a more informative response or raise an appropriate exception if the movie is not found.